tests/unit/amo/components/TestAddonVersionCard.js

```diff
@@ -255,12 +280,26 @@ describe(__filename, () => {
     expect(root.find(AddonInstallError)).toHaveProp('error', error);
   });
 
-  it('passes an add-on to AddonCompatibilityError', () => {
+  it('does not render an AddonInstallError if there is no version', () => {
+    const root = render({ version: null });
```

Comment:
Doesn't this test need to begin by setting up an error? Something like: `store.dispatch(setInstallError({ error, guid }));`. I can't get the test to fail as is.